scripts/check-health.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# assert on the health of a running container
+IS_SYNCING=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.isSyncing')
```

Comment:
You could save the output to a variable and then pass it to jq, currently this is 3 requests per health check.

Something like

```bash
HEALTH=$(curl --silent http://127.0.0.1:9933/health)
jq -r '.isSyncing' <<< $HEALTH
```